dashboard/src/modules/components/ProfileComponent/index.jsx

```diff
@@ -217,7 +215,7 @@ const ProfileComponent = () => {
                     <TextContent>
                       <span>Account creation Date</span>
                       <Text component={TextVariants.h4}>
-                        {formatData(user?.registered_on)}
+                        {formatDate(user?.registered_on)}                       
```

Comment:
Do you have a way to automatically clean trailing whitespace in you editor?

Reply:
Yes, white space is removed.